### ISSUE TITLE
next-css plugin doesn't work

### ISSUE THREAD

**Issue by awinogradov** [zeit/next-plugins#268]
So, after:
- `npm install --save @zeit/next-css`
- echo '`const withCSS = require('@zeit/next-css'); module.exports = withCSS();`' > `next.config.js`

`./style.css`
``` css
body {
    background: #000;
}

.Page {
    color: red;
}
```

`./pages/index.js`
``` js
import '../style.css';

export default () => <div className="Page">Welcome to next.js!</div>
```

- `next`

No CSS on the page.

**Comment by kachkaev** [comment]
Hey @awinogradov! Could you please share your example as a reproducable repo? What happens if you try canary versions?

**Comment by awinogradov** [comment]
Sure, https://github.com/awinogradov/next-css-example

I haven't tried canary versions.

**Comment by timneutkens** [comment]
99% sure this is fixed with Next 7 + next-css 1.0

**Comment by ITforHome** [comment]
i use Next 7.0.0 and Next-CSS 1.0.1 and i already have the same issue as 'awinogradov' Can you also help me to fix the problem? Many Thanks
